engine/runtime/src/main/java/org/enso/interpreter/caches/Cache.java

```diff
@@ -158,6 +190,8 @@ private boolean saveCacheTo(
                 + "] to ["
                 + toMaskedPath(parentPath).applyMasking()
                 + "].");
+        recordCacheEvent(() -> new CacheEvent.Save(logName, bytesToWrite.length));
+        memoryArena.close();
```

Comment:
- Error!
- You need to close the mmapped file - e.g. the arena first and only then `writeBytesTo`

Reply:
Done in 46b42e5941099dc9e85d220f06088c8f4b2d03ab